Sources/News.php

```diff
@@ -789,7 +789,7 @@ function getXmlMembers($xml_format)
 		SELECT id_member, member_name, real_name, date_registered, last_login
 		FROM {db_prefix}members
 		ORDER BY id_member DESC
-		LIMIT {int:limit} OFFSET {int:offset}',
+		LIMIT {int:limit}, {int:offset}',
```

Comment:
when you use the syntax without keyword offset, you to write the offset on the left side and limit on the right side.

An keep in mind this syntax is mysql solution, the sql standard is the with the keyword offset.